README.ios.md

```diff
@@ -0,0 +1,43 @@
+# Glean SDK
+
+## Documentation
+
+All documentation is available online:
+
+## [The Glean SDK Book][book]
+
+## Overview
+
+Refer to the documentation for [using and developing the Glean SDK][book].
+
+For an overview of Glean beyond just the SDK, see the [section in the Firefox data docs](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
+
+**Note: The Glean SDK requires at least [Rust 1.34.2](https://blog.rust-lang.org/2019/05/14/Rust-1.34.2.html). Older versions are untested.**
+
+## Contact
+
+To contact us you can:
+
+* Find us on the Mozilla Slack in *#glean*, on [Mozilla IRC][mozirc] in *#telemetry*.
+* To report issues or request changes, file a bug in [Bugzilla in Data Platform & Tools :: Glean: SDK][newbugzilla].
+* Send an email to *glean-team@mozilla.com*.
+* The Glean Core team is: *:dexter*, *:janerik*, *:mdroettboom*, *:gfritzsche*
```

Comment:
Me too?

Reply:
Yes! This should be fixed in the main readme as well :) I think this might need to be changed to the whole team in Q4, given project FOG.